dali/operators/generic/slice/slice_base.cc

```diff
@@ -74,20 +74,33 @@ template <typename OutputType, typename InputType, int Dims>
 void SliceBaseCpu<OutputType, InputType, Dims>::RunImpl(workspace_t<CPUBackend> &ws) {
   const auto &input = ws.template InputRef<CPUBackend>(0);
   auto &output = ws.template OutputRef<CPUBackend>(0);
+  output.SetLayout(input.GetLayout());
```

Comment:
:+1:

Reply:
It was there, but at the end of the function.